source/jormungandr/jormungandr/interfaces/v1/Uri.py

```diff
@@ -113,6 +113,16 @@ def __init__(self, is_collection, collection, *args, **kwargs):
             'the moment of the request. It will mainly change the output '
             'of the disruptions.',
         )
+        parser.add_argument(
+            "data_freshness",
+            help="Define the freshness of data to use to compute vehicle_journeys "
+            "along with parameters &since and/or &until .\n"
+            "When using `&data_freshness=base_schedule` you get original vehicle_journeys "
+            "where as when using `&data_freshness=realtime` you get vehicle_journeys "
+            "added or modified by realtime.",
+            type=OptionValue(['base_schedule', 'adapted_schedule', 'realtime']),
+            default='base_schedule',
+        )
```

Comment:
Adding the parameter in this class adds it in every classes inheriting it. In some of them, 'data_freshness' doesn't make much sense. Is it worth it?
If the goal here is to fix /disruptions and /vehicle_journeys, why not add it only in these classes?

Reply:
It should be defined with `since` and  until` to be consistent (so here)